### ISSUE TITLE
Improve accessibility of Malwarebytes Anti-Malware with NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#856]
**Reported by elliott94 on 2010-08-24 14:28**
Firstly, apologies for the ticket description - I couldn't think of anything else to write.

Currently, users of the popular Malwarebytes Anti-Malware application are unable to update the database deffinitions using NVDA; the tabbing system that is used by the program can be detected using object navigation, however it is unable to move between tabs.
Also, in addition to this, the infections list isn't read when the scan completes (if any are found) and filenames arn't read when the program is scanning the system.

**Comment by leonardder** [comment]
CC @JosephSL @Elliot94 @JCSTeh @Brian1Gaff

**Comment by ehollig** [comment]
See #4216 for additional information on this issue.

**Comment by Adriani90** [comment]
This is not reproducible anymore, I am using the newest malware Bytes and it works as expected.

**Comment by ehollig** [comment]
Thank you @Adriani90. Closing as worksforme
